internal/packagemanager/packagemanager.go

```diff
@@ -212,6 +197,27 @@ func (pm *DistroPackageManager) GetSSMPackage() artifact.Package {
 	)
 }
 
+func (pm *DistroPackageManager) getCACertsPackage() artifact.Package {
```

Comment:
nit: in general go tends to omit "get" unless the method actually "gets" something externally, like calling a GET api.
```suggestion
func (pm *DistroPackageManager) caCertsPackage() artifact.Package {
```